generator/src/test/java/com/scottlogic/deg/generator/cucumber/ShorterThan.feature

```diff
@@ -7,11 +7,15 @@ Background:
 
 Scenario: Running a 'shorterThan' request using a number to specify a the length of a generated string should be successful
      Given foo is shorter than 5
-       And foo is matching regex /[x]{0,5}/
-     Then the following data should not be included in what is generated:
+       And foo is matching regex /[x]{1,5}/
+       And the generator can generate at most 5 rows
+     Then the following data should be generated:
        | foo      |
-       | "xxxxx"  |
-       | "xxxxxx" |
+       | null     |
```

Comment:
is null supposed to be produced if we are producing a regex string between 1 and 5 chars?

Reply:
The current decision is that null is always created unless the profile specifically states `not isNull`

Reply:
okay, thanks.